pkg/autoscaler/metrics/collector_test.go

```diff
@@ -399,13 +442,18 @@ func scraperFactory(scraper StatsScraper, err error) StatsScraperFactory {
 
 type testScraper struct {
 	s   func() (Stat, error)
+	r   func() ([]string, error)
 	url string
 }
 
 func (s *testScraper) Scrape(time.Duration) (Stat, error) {
 	return s.s()
 }
 
+func (s *testScraper) ScrapeForRemoval(ready, desired int) ([]string, error) {
+	return s.r()
```

Comment:
same here `ready, desired` not used in function

Reply:
same as above. this is a fake struct implementing an interface.